mantidproject/mantid
# Add key delete event for workspace dock

This is now localised to just work in the dock itself.

Description of work.

**To test:**

See reproduction instructions in bug report. Additionally check:
 - the "Delete" button still works
 - deleting masks with the delete key works as well.

Fixes #18976 .

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
